.tool/genheader.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"os"
+	"strings"
+	"text/template"
+
+	specs "github.com/opencontainers/image-spec/specs-go"
+)
+
+var headerTemplate = template.Must(template.New("gen").Parse(`<title>image-spec {{.}}</title>
+<base href="https://raw.githubusercontent.com/opencontainers/image-spec/{{.}}/">`))
+
+func main() {
+	version := specs.Version
+	if strings.HasSuffix(specs.Version, "-dev") {
+		version = "master"
```

Comment:
> Why not just keep the *-dev version?

Ah, because it won't work for `<base>`.  Maybe use `specs.Version` directly in `<title>`, and then use the HEAD commit hash in `<base>` when `specs.Version` ends with `-dev`.